### ISSUE TITLE
Unable to use 'nest' filter on Windows

### ISSUE THREAD

**Issue by marcelo23** [fluent/fluent-bit#1743]
## Bug Report

**Describe the bug**

Unable to use 'nest' plugin on latest Windows Fluent Bit version 1.3.2 (from the zip distribution). Getting error 'Invalid configuration value at FILTER.Name' when 'nest' plugin is defined in configuration. Thus unable to produce a nested JSON file.

**To Reproduce**
- main.conf:
```
[SERVICE]
    Parsers_File ../conf/parsers.conf

[INPUT]
    Name tail
    Path C:/dev/tools/td-agent-bit-1.3.2-1.AMD64/core/input/app-*.log
    Tag cp_core
    Multiline On
    Parser_Firstline my-awesome-parser
    Parser_1 my-awesome-multiline-parser

[FILTER]
    Name modify
    Match *
    Rename message header.eventMessage
    Rename severity header.severityLevel
    Add header.eventType -

[FILTER]
    Name          nest
    Match         *
    Operation     nest
    Wildcard      header.*
    Nest_under    header
    Remove_prefix header.

[OUTPUT]
    Name file
    Match *
    Path C:/dev/tools/td-agent-bit-1.3.2-1.AMD64/core/output/output.txt
```

parsers.conf:
```
[PARSER]
    Name my-awesome-parser
    Format regex
    Regex ^(?<timestamp>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}) (?<severity>[A-Z]{4,7}) \[(?<class>[a-zA-Z<>.:0-9]+)\] (?<thread>\([a-zA-Z0-9\s-]+\))?\s?\-?\s*(?<message>[^\b]*)
    Time_Key timestamp
    Time_Format %Y-%m-%d %H:%M:%S,%s
    Time_Keep On

[PARSER]
    Name my-awesome-multiline-parser
    Format regex
    Regex ^(?!<message>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3} )
```

- Example log message:
```
2019-11-13 05:42:10,111 ERROR [SequenceHandlerManager] (Custom Pool-pool-4-thread-1) 2  contracts found with modification time 2016-10-25 12:16:00.0
```

- Steps to reproduce the problem:
1. Run fluent-bit: C:\dev\tools\td-agent-bit-1.3.2-1.AMD64\bin>fluent-bit.exe -c ../core/conf/core.conf
2. Check console output for any errors upon startup, mentioned message appears, 'nest' plugin is not initialized:
```
C:\dev\tools\td-agent-bit-1.3.2-1.AMD64\bin>fluent-bit.exe -c ../core/conf/core.conf
Fluent Bit v1.3.2
Copyright (C) Treasure Data

Invalid configuration value at FILTER.Name
[2019/11/15 14:14:20] [debug] [storage] [cio stream] new stream registered: tail.0
[2019/11/15 14:14:20] [ info] [storage] initializing...
[2019/11/15 14:14:20] [ info] [storage] in-memory
[2019/11/15 14:14:20] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
[2019/11/15 14:14:20] [ info] [engine] started (pid=22908)
etc etc ......
.........
```

**Expected behavior**

Expected behaviour is no errors in the console and JSON output produced is nested based on configuration

**Screenshots**

**Your Environment**

* Version used: 1.3.2-1.AMD64 for Windows
* Environment name and version (e.g. Kubernetes? What version?):
* Operating System and version: Windows 7 x64 Enterprise SP1
* Filters and plugins: tail, modify, nest, file

**Comment by fujimotos** [comment]
> Unable to use 'nest' plugin on latest Windows Fluent Bit version 1.3.2 (from the zip distribution). Getting error 'Invalid configuration value at FILTER.Name' when 'nest' plugin is defined in configuration. Thus unable to produce a nested JSON file.

I spent some time working on this today and posted a fix to #1745.
This issue should be resolved when the PR gets merged.

if someone wants to try it earlier, please use the following installer:

https://ci.appveyor.com/project/fluent/fluent-bit-2e87g/builds/28926020/job/ffm2ayu1o5ljq7j6/artifacts

**Comment by edsiper** [comment]
fixed by @fujimotos on GIT Master
